### PR TITLE
Updated csv export

### PR DESCRIPTION
**Pull request by siowhoi**
1. Added new export modal
2. Export modal will get all records of the entity

Screenshot 2024-01-25 at 10 29 29 am

### CODE REVIEW

File: src/packages/admin-ui-components/src/utils/utils.ts [hunk 1]
```diff
@@ -6,3 +8,18 @@ export const isNumeric = (item: unknown): boolean => {
...
 	return false;
 };
+
+export const getOrderByQuery = (sort?: SortField[]) => ({
+	...(sort
+		? sort.reduce((acc, { field, direction }) => ({ ...acc, [field]: direction }), {})
+		: { id: 'ASC' }),
+});
+
+export const getAndFiltersQuery = (filters: FieldFilter) => {
```
Comment (steve-keep) [inline src/packages/admin-ui-components/src/utils/utils.ts:18]: I wonder if a better name for this is `wrapFilterWithAndOperator`

File: src/packages/admin-ui-components/src/export-modal/styles.module.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.exportContainer {
+	background: var(--detail-bg-color);
+	border: 1px solid rgba(237, 232, 242, 0.1);
```
Comment (steve-keep) [inline src/packages/admin-ui-components/src/export-modal/styles.module.css:3]: should we add this color as a var?

File: src/packages/admin-ui-components/src/modal/styles.module.css [hunk 3]
```diff
@@ -1,5 +1,6 @@
 .overlay {
-	background-color: rgb(0 0 0 / 30%);
+	z-index: 999;
+	background-color: rgba(16, 10, 28, 0.6);
```
Comment (steve-keep) [inline src/packages/admin-ui-components/src/modal/styles.module.css:3]: should we add this color as a var?

**Review by steve-keep — APPROVED** [review]
Looks good, a couple of tiny things to check if you have time.

File: src/packages/admin-ui-components/src/utils/utils.ts (hunk 1 above)
Comment (siowhoi) [reply src/packages/admin-ui-components/src/utils/utils.ts]: The function has now been renamed to `wrapFilterWithAndOperator `.

File: src/packages/admin-ui-components/src/export-modal/styles.module.css (hunk 2 above)
Comment (siowhoi) [reply src/packages/admin-ui-components/src/export-modal/styles.module.css]: Fixed.

File: src/packages/admin-ui-components/src/modal/styles.module.css (hunk 3 above)
Comment (siowhoi) [reply src/packages/admin-ui-components/src/modal/styles.module.css:3]: Fixed.

**Pull request merged**